bedrock/base/templates/includes/canonical-url.html

```diff
@@ -10,7 +10,7 @@
         {% if code == 'en-US' -%}
         <link rel="alternate" hreflang="en" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English">
         <link rel="alternate" hreflang="en-CA" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English (Canada)">
-        {% else -%}
+        {% elif code not in ['ast', 'cak', 'dsb', 'hsb', 'kab', 'lij', 'son'] -%}{#- Bug 1364470: drop locales not supported by Google -#}
```

Comment:
I think we should keep this list in settings. It feels kindof hidden here. Settings are already in the template context, so it should be a simple addition to `settings/base.py` and change this to `elif code not in settings.LANGS_NON_GOOGLE` (or whatever the setting name ends up being).